ethcore/src/block.rs

```diff
@@ -374,7 +375,7 @@ impl<'x> OpenBlock<'x> {
 		s.block.header.set_uncles_hash(keccak(&uncle_bytes));
 		s.block.header.set_state_root(s.block.state.root().clone());
 		s.block.header.set_receipts_root(ordered_trie_root(s.block.receipts.iter().map(|r| r.rlp_bytes())));
-		s.block.header.set_log_bloom(s.block.receipts.iter().fold(Bloom::zero(), |mut b, r| {b = &b | &r.log_bloom; b})); //TODO: use |= operator
+		s.block.header.set_log_bloom(s.block.receipts.iter().fold(Bloom::zero(), |b, r| &b | &r.log_bloom ));
```

Comment:
this will be even more efficient if you use [accrue_bloom](https://docs.rs/ethbloom/0.5.0/ethbloom/struct.Bloom.html#method.accrue_bloom) method as it modifying bloom inplace without allocating 256 bytes for every loop iteration.

Reply:
logged an issue for that :)